test/sequential/test-cpu-prof.js

```diff
@@ -3,8 +3,7 @@
 // This tests that --cpu-prof and --cpu-prof-path works.
 
 const common = require('../common');
-if (process.features.debug &&
-  process.config.variables.node_code_cache_path === 'yes') {
+if (process.features.debug && process.config.variables.node_code_cache_path) {
```

Comment:
We could also add something in the `native_module` binding to inform the JS land about the availability of the embedded code cache, which could be computed with some static bool variable set in both the stub and the generated cache code, that probably will more reliable than a configure variable.

Reply:
I've been thinking on the same lines. And expose it as a `process.features` field.